MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+global-include *.rst
+include LICENSE
+include COPYING
+graft docs
+graft tests/unit
+include examples/haas.cfg
```

Comment:
Why just this from examples? We should be bundling up everything that belongs in a source tarball, i.e. everything that's actually checked in to the repo.